src/less/variables-swedbankpay.less

```
@@ -215,4 +217,9 @@
     --btn-disabled-text: var(--text-disabled);
     --btn-bg-disabled: rgba(212, 196, 188, 0.3);
     --btn-active-color: var(--brand-turquoise-light);
+
+    //FORM VALUES
+    --input-border-color: var(--brown-soft);
+    --input-border-focus-color: var(--brown-solid);
+    --input-border-color-success: var(--brand-turquoise);
```

I'd suggest renaming it following the same pattern as the variable before:
```suggestion
    --input-border-success-color: var(--brand-turquoise);
```